app/amp_start.py

```diff
@@ -13,25 +13,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import urlparse
 import start
+import urlparse
```

Comment:
Looks like urlparse and utils are no longer used. Can you remove them?

Reply:
done.